examples/resources/leanspace_processors.tf

```diff
@@ -0,0 +1,11 @@
+variable "processor_path" {
+  type        = string
+  description = "The path to the processor's source code file (a .jar)."
+}
+
+resource "leanspace_processors" "test_create_processor" {
+  file_path                            = var.processor_path
+  version                              = "v1.0"
```

Comment:
Nit: no need for v it seems to be already displayed on the UI

Reply:
Done